src/ext/browserUtils.js

```diff
@@ -127,7 +145,11 @@ function isCookieForTab(hrefs, cookieDomain) {
     return false;
 }
 
-// Tor seems to have an object here whereas chrome/firefox just have an id
```

Comment:
Would it make sense to move this comment inside the function such that the justification for this function is maintained?